modular_skyrat/modules/gun_cargo/code/armament_datums/izhevsk.dm

```diff
@@ -43,6 +43,11 @@
 	item_type = /obj/item/gun/ballistic/rifle/boltaction
 	lower_cost = CARGO_CRATE_VALUE * 5
 	upper_cost = CARGO_CRATE_VALUE * 6
+	
+/datum/armament_entry/cargo_gun/izhevsk/rifle/mosin/brandnew
+	item_type = /obj/item/gun/ballistic/rifle/boltaction/brand_new
```

Comment:
There's literally no difference between the newer (brand-new) Mosin and the older (OG boltaction) one in terms of stats and sprites, except for the fact that the earlier can't be sawn-off. I'd advise either making a unique subtype with some "cewl" feature (like semi-auto/scoped Mosin? :flushed:) or not adding the brand-new one outright in favor of something else. A Nagant revolver possibly - it's hardly seen anywhere.

Reply:
Nagant revolver existed but was removed, please don't readd.

Reply:
> There's literally no difference between the newer (brand-new) Mosin and the older (OG boltaction) one in terms of stats and sprites, except for the fact that the earlier can't be sawn-off. I'd advise either making a unique subtype with some "cewl" feature (like semi-auto/scoped Mosin? 😳) or not adding the brand-new one outright in favor of something else. A Nagant revolver possibly - it's hardly seen anywhere.

there is, bolt speed, brand_new's have a bolt delay of about 0.5s, while the default mosin has to wait 0.8 (essentially 1 second) between bolting actions

Reply:
> there is, bolt speed, brand_new's have a bolt delay of about 0.5s, while the default mosin has to wait 0.9 (essentially 1 second) between bolting (cycling) actions
Oh epic.